lua/session_manager/utils.lua

```diff
@@ -126,12 +126,15 @@ function utils.get_sessions()
   end
   table.sort(sessions, function(a, b) return a.timestamp > b.timestamp end)
 
-  -- If we are in a session already, show the penultimate saved session on top.
-  -- If we are not inside a session, show the latest saved session on top.
   if utils.is_session then
+    -- If we are in a session already, don't list the current session.
     local cwd = vim.loop.cwd()
-    if #sessions >= 2 and cwd and config.dir_to_session_filename(cwd).filename == sessions[1].filename then
-      sessions[1], sessions[2] = sessions[2], sessions[1]
+    local is_current_session = cwd and config.dir_to_session_filename(cwd).filename == sessions[1].filename
+    if is_current_session then table.remove(sessions, 1) end
+
+    -- If no sessions to list, show a notification.
+    if (is_current_session and #sessions == 0) or sessions == nil then
+        vim.notify("The only available session is your current session. Nothing to show yet.", vim.log.levels.INFO)
```

Comment:
Also should we return here?
```suggestion
        vim.notify("The only available session is your current session. Nothing to select from.", vim.log.levels.INFO)
```

Reply:
Change applyied on https://github.com/Shatur/neovim-session-manager/commit/379fb613d6a9d8e9925dab869a82d52ff8bfd8b8